### PR TITLE
Remove --last-dump-id flag to simplify dumps code

### PR DESCRIPTION
**Pull request by amCap1712**
We have decided to simplify the dumps code, by default a dump will always add an entry to data_dump table. In case something goes wrong, we will pass in the `--dump-id` manually.

### CODE REVIEW

File: admin/create-dumps.sh [hunk 1]
```diff
@@ -77,6 +77,9 @@ if [ -z $DUMP_BASE_DIR ]; then
 fi
 
 DUMP_TYPE="${1:-full}"
+# consume dump type argument so that we can pass the remaining arguments to
+# the python dump manager script
```
Comment (alastair) [inline admin/create-dumps.sh:81]: in this case it'd be good to have some kind of documentation for this script that explains that we're able to do this. Perhaps just comments at the top of the file like we do in develop.sh and test.,sh is enough for now

**Review by alastair — COMMENTED** [review]
I don't understand the behaviour of this flag, but it sounds like you've gone through it a few times and worked out what was going on, so I'm happy to go with your suggestion 👍

**Review by mayhem — APPROVED** [review]
Yeah, for simplification!

File: admin/create-dumps.sh (hunk 1 above)
Comment (amCap1712) [reply admin/create-dumps.sh:87]: Added a few line on script usage at top.

**Pull request merged**